autogpt_platform/backend/backend/data/block.py

```diff
@@ -272,8 +272,6 @@ def execute(self, input_data: BlockInput, **kwargs) -> BlockOutput:
         for output_name, output_data in self.run(
             self.input_schema(**input_data), **kwargs
         ):
-            if "error" in output_name:
```

Comment:
Non-refactoring change: 
We will still allow manual yielding of `error` that won't break the dlow.